proxy/src/graphql/schema.rs

```diff
@@ -510,6 +510,10 @@ impl coco::Commit {
         &self.message
     }
 
+    fn committer(&self) -> &coco::Person {
+        &self.committer
+    }
+
     fn committer_time(&self) -> String {
         self.committer_time.seconds().to_string()
     }
```

Comment:
Should I remove this now? Only problem is that the time won't be converted to seconds anymore.

Reply:
I think you can't remove this as juniper won't know how to convert that type.